eZ/Publish/SPI/Search/Indexer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Search;
+
+/**
+ * Indexer for handlers that index stuff.
+ */
+interface Indexer
```

Comment:
True, it will become wrong for the new signal. New slot to handle that signal will know about it, so that's fine.